functests/1_performance/cpu_management.go

```diff
@@ -282,32 +307,38 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			output, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
 			Expect(err).ToNot(HaveOccurred())
 
-			cpu, err := strconv.Atoi(output)
+			cpus, err := cpuset.Parse(output)
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Getting the CPU scheduling flags")
-			flags, err := getCPUSchedulingDomainFlags(cpu)
+			flags, err := getCPUsSchedulingDomainFlags()
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Verifying that the CPU load balancing was disabled")
-			Expect(len(flags)).To(Equal(len(defaultFlags)))
-			// the CPU flags should be almost the same except the LSB that should be disabled
-			for i := range flags {
-				Expect(flags[i]).To(Equal(defaultFlags[i] - 1))
+			for _, cpu := range cpus.ToSlice() {
+				Expect(len(flags[cpu])).To(Equal(len(defaultFlags[cpu])))
+				// the CPU flags should be almost the same except the LSB that should be disabled
+				// see https://github.com/torvalds/linux/blob/0fe5f9ca223573167c4c4156903d751d2c8e160e/include/linux/sched/topology.h#L14
```

Comment:
Thank you, this really helps! We should always link the source info in the code!